setup.py

```diff
@@ -387,7 +387,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
     # It contains a number of potential breaking changes but none of them looks breaking our use
     # https://docs.pytest.org/en/latest/changelog.html#pytest-7-0-0-2022-02-03
     # TODO: upgrade it and remove the limit
-    "pytest~=6.0",
+    "pytest~=7.2.1",
```

Comment:
I think if main idea remove upper bound limitations and better use `pytest>=6.0` or `pytest>=7.0` (if we intend to use features which added only in 7.0 branch)

Maybe it is not a case of `pytest`.

Reply:
make sense, done